app/views/quotes/_quote.html.erb

```diff
@@ -1,14 +1,18 @@
 <%# app/views/quotes/_quote.html.erb %>
 
-<div class="quote">
-  <%= link_to quote.name, quote_path(quote) %>
-  <div class="quote__actions">
-    <%= button_to "Delete",
-                  quote_path(quote),
-                  method: :delete,
+<%= turbo_frame_tag "quote_#{quote.id}" do %>
```

Comment:
1.- In the tutorial, it is mentioned that we need to add the ID 'quote' to get this ID with the appropriate 'quote' and replace it with the same ID already updated.